admin/spec/service/check-form.service.spec.js

```diff
@@ -129,4 +129,21 @@ describe('check-form.service', () => {
       done()
     })
   })
+
+  describe('#unassignedCheckFormsFromCheckWindows()', () => {
+    beforeEach(() => {
+      service = proxyquire('../../services/check-form.service', {
+        '../../services/check-form.service': checkFormService
```

Comment:
This looks ... odd.  It has itself as a dependency?

Reply:
That doesn't seem like a case where you would use proxyquire indeed, simple require would do.